docs/attribution_parameterization.md

```diff
@@ -0,0 +1,117 @@
+## Attribution Parameterization 
+
+Attributions for different models and use cases can range from simple to more complex. This will help see how to set the various parameters to get what you need.
+
+### Basic Definitions and Terminology
+
+**What is a tensor?**
+A tensor is a multidimensional object that can be model inputs, or layer activations.
+
+**What is a layer?**
+A layer is a set of neurons that can be thought of as a function on input tensors. Layer inputs are tensors. Layer outputs are modified tensors.
+
+**What are anchors?**
+Anchors are ways of specifying which tensors you want. You may want the input tensor of a layer, or the output tensor of a layer. 
+
+Eg: Say you have a concat layer and you want to explain the 2 concatenated tensors. The concat operation is not usually a layer tracked by the model. If you try the 'in' anchor of the layer after the operation, you get a single tensor with all the information you need.
+
+**What is a Quantity of Interest (QoI)?**
+A QoI is a scalar number that is being explained. 
+
+Eg: With saliency maps, you get `dx/dy` (i.e the effect of input on output). `y` in this case is the QoI scalar. It is usually the output of a neuron, but could be a sum of multiple neurons.
+
+**What is an attribution?**
```

Comment:
Is there somewhere we can explain the relationship between the DoI and attribution cut?